api/schema_handler.go

```diff
@@ -128,6 +131,10 @@ func (handler *SchemaHandler) GetTable(w http.ResponseWriter, r *http.Request) {
 //    default: errorResponse
 //        200: noContentResponse
 func (handler *SchemaHandler) AddTable(w http.ResponseWriter, r *http.Request) {
+	if handler.isClusterMode {
```

Comment:
we can modify Register to just register the read/Get calls

Reply:
i'm afraid that would confuse user with 404 error. 405 (not allowed) may make more sense?

Reply:
hmm... i think we can use router.Methods() to control the methods